### PR TITLE
Updated to new Flutter 3.0.4 (with null safety)

### PR DESCRIPTION
**Pull request by ValeriusGC**
Hi! Nice sample, i updated it.

### CODE REVIEW

File: lib/presentation/notifiers/timezones_notifier.dart [hunk 1]
```diff
@@ -7,8 +7,12 @@ class TimezoneNotifier extends StateNotifier<List<String>> {
...
   }
 
-  _init() async {
-    state = await WorldTimeApi.getTimeZones() ?? [];
+  _init() {
+    print('TimezoneNotifier._init.1: $state');
+    Future.delayed(Duration(milliseconds: 1000)).then((_) async {
+      state = await WorldTimeApi.getTimeZones() ?? [];
+      print('TimezoneNotifier._init.2: $state');
+    });
```
Comment (lohanidamodar) [inline lib/presentation/notifiers/timezones_notifier.dart]: Why was the delay required?

**Review by lohanidamodar — COMMENTED** [review]
Looks good, just left a query

File: lib/presentation/notifiers/timezones_notifier.dart (hunk 1 above)
Comment (ValeriusGC) [reply lib/presentation/notifiers/timezones_notifier.dart]: There was strange behavior then, API didn't give any data without this delay. I checked now - everything is ok, delay is removed.

**Comment by ValeriusGC** [comment]
All checked, everything works.
```
Flutter 3.3.4 • channel stable • https://github.com/flutter/flutter.git
Framework • revision eb6d86ee27 (6 weeks ago) • 2022-10-04 22:31:45 -0700
Engine • revision c08d7d5efc
Tools • Dart 2.18.2 • DevTools 2.15.0
```

**Comment by lohanidamodar** [comment]
@ValeriusGC Why used hooks? I wanted to keep the example simple without the hooks. Merging it into a new Flutter 3.x branch

**Comment by lohanidamodar** [comment]
Thanks for the contribution @ValeriusGC merged to new branch and updated Readme in the master to point to the branch

**Comment by ValeriusGC** [comment]
> @ValeriusGC Why used hooks? I wanted to keep the example simple without the hooks. Merging it into a new Flutter 3.x branch

Sorry for hooks.

**Pull request merged**